new-lamassu-admin/src/pages/Machines/Machines.js

```diff
@@ -77,10 +76,10 @@ const Machines = () => {
                 </Label3>
               </Link>
               <TL2 noMargin className={classes.subtitle}>
-                {selectedMachine}
+                {machine.name}
```

Comment:
above where we declared const machine, we can also do now:

const machineName = R.path(['name'])(machine) ?? null